sd_meh/merge.py

```diff
@@ -59,8 +59,14 @@ def fix_model(model: Dict) -> Dict:
     return fix_clip(model)
 
 
-def load_sd_model(model_path: os.PathLike, device: str = "cpu") -> SDModel:
-    return SDModel(model_path, device).load_model()
+def load_sd_model(model: os.PathLike | str | Dict, device: str = "cpu") -> Dict:
+    if isinstance(model, Dict):
```

Comment:
It's to be able to call merge multiple times in a row without having to pass by the filesystem.

Didn't know about pathlike, will remove the str case

Reply:
Well my python interpreter tells me otherwise for `PathLike`:
```
>>> isinstance('', os.PathLike)
False
```

Reply:
This is one example use case:

```python
import sd_meh.merge as meh_merge
import pathlib


model_files = [f for f in pathlib.Path('path to list of models...').glob('*.safetensors')]
merged = model_files[0]

for merge_i, model_a in enumerate(model_files[1:]):
    alpha = 1 / (merge_i + 1)

    models = {"model_a": model_a, "model_b": merged}
    bases = {"alpha": alpha}
    weights = {"alpha": [alpha] * 25}

    print(f'iteration {merge_i}')
    merged = meh_merge.merge_models(
        models, weights, bases, "weighted_sum", 16
    )

meh_merge.save_model(merged, r'final model path', 'safetensors')
```

Reply:
Let's say I have a folder with 25 checkpoints. I don't want to have to wait for filesystem writes/reads. On my ssd it is already slow, it would be painful on a hdd.

Reply:
maybe add an `in_place` flag to allow users to save memory?

Reply:
In any case, I will make this change in a different PR then, as just allowing to pass a dict will give confusing results.